core/src/main/java/nu/yona/server/messaging/service/DisclosureRequestMessageDto.java

```diff
@@ -177,16 +178,14 @@ private DisclosureRequestMessage updateMessageStatus(DisclosureRequestMessage di
 		private void sendResponseMessageToRequestingUser(UserDto respondingUser, DisclosureRequestMessage requestMessageEntity,
 				String message)
 		{
-			MessageDestinationDto messageDestination = userAnonymizedService.getUserAnonymized(requestMessageEntity
-					.getRelatedUserAnonymizedId()
-					.orElseThrow(() -> new IllegalStateException(
-							"Message with ID " + requestMessageEntity.getId() + " does not have a related user anonymized ID")))
-					.getAnonymousDestination();
-			assert messageDestination != null;
+			UserAnonymizedDto toUser = userAnonymizedService.getUserAnonymized(
+					requestMessageEntity.getRelatedUserAnonymizedId().orElseThrow(() -> new IllegalStateException(
+							"Message with ID " + requestMessageEntity.getId() + " does not have a related user anonymized ID")));
+			assert toUser != null;
```

Comment:
That is not completely true, and I didn't change this code, but will remove the assert because there is also an `orElseThrow` inside `getUserAnonymized`